examples/detection/detection.py

```diff
@@ -162,7 +207,5 @@ async def job_request_cb(job_request: agents.JobRequest):
             auto_disconnect=agents.AutoDisconnect.DEFAULT,
         )
 
-    worker = agents.Worker(
-        request_handler=job_request_cb, worker_type=agents.JobType.JT_ROOM
-    )
+    worker = agents.Worker(request_handler=job_request_cb)
```

Comment:
nit: you can directly do that
```suggestion
    worker = agents.Worker(job_request_cb)
```